data/models/suggestion.py

```diff
@@ -89,6 +105,11 @@ class Meta:
 
 
 class Suggestion(models.Model):
+
+    class Meta:
+        verbose_name = "1️⃣ Suggestion"
+        verbose_name_plural = "1️⃣ Suggestions"
```

Comment:
```suggestion
        verbose_name = "1️⃣ Suggestion"
```

facultatif ici car le S sera automatiquement ajouté à la fin